testsuite/features/secondary/srv_restart_postgres.feature

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the postgres service
+
+  Scenario: Restart the postgres
+    Given I am authorized for the "Admin" section
+    When I restart "postgresql.service" service on "server"
+    # taskomatic needs 60s to reconnect
+    And I wait for "60" seconds
```

Comment:
Pleas instead of waiting always 60 seconds, better use this step:

`And service "postgresql" is active on "server"`

Reply:
I agree, but was wondering if it is really enough to wait until the PostgreSQL service is active again, or if we really need to wait longer for Taskomatik to reconnect as Vladimir wrote.

Reply:
Yes, if we see a message in the log, we can wait for it. I will do some test around this on my test server. Resolving this conversation. We have your comment below.